azure-pipelines.yml

```diff
@@ -40,7 +40,7 @@ stages:
       DIFF_COVER_WORKING_DIRECTORY: $(System.DefaultWorkingDirectory)/sonic-gnmi
 
     container:
-      image: sonicdev-microsoft.azurecr.io:443/sonic-slave-buster:latest
+      image: sonicdev-microsoft.azurecr.io:443/sonic-slave-bullseye:latest
```

Comment:
>bullseye [](http://example.com/codeflow?start=59&length=8)

Let's separate irrelevant changes such as "bullseye" to another PR and focus "native mode gnmi service" in this PR.

If you mix, it's challenge to revert anything in case.